src/screens/InheritanceToolsAndTips/InheritanceTool.tsx

```diff
@@ -26,6 +34,8 @@ import { LocalizationContext } from 'src/context/Localization/LocContext';
 function InheritanceTool({ navigation }) {
   const dispatch = useAppDispatch();
   const { plan } = usePlan();
+  const isHodlerAndDiamondHand = plan === 'DIAMOND HANDS' || plan === 'HODLER';
```

Comment:
Why are we still using hardcoded text for comparison? Please handle that comparison elegantly